Core/clim-basic/recording.lisp

```diff
@@ -1032,15 +1032,6 @@ were added."
   (if-supplied (text-style)
     (text-style-equalp (slot-value record 'text-style) text-style)))
 
-(defmethod replay-output-record :around
-    ((record gs-transformation-mixin) stream &optional region x-offset y-offset)
-  (declare (ignore region x-offset y-offset))
-  (with-drawing-options (stream :transformation (graphics-state-transformation record))
-    (call-next-method)))
```

Comment:
how is this breaking the dialog? I'm still convinced, it is the right thing to do. if some other record type doesn't want with-drawing-options :transformation, then it shouldn't inherit from gs-transformation-mixin.

And this is around replay-output-record on draw-text-output-record, which inherits from gs-transformation-mixin, so for this kind of records it should make a difference (so this should be functionally equivalent to the reverted version).